src/config/networks/binance.ts

```diff
@@ -0,0 +1,64 @@
+import BnbLogo from 'src/config/assets/token_bnb.svg'
+import { EnvironmentSettings, ETHEREUM_NETWORK, FEATURES, NetworkConfig, WALLETS } from 'src/config/networks/network.d'
+
+const baseConfig: EnvironmentSettings = {
+  clientGatewayUrl: 'https://safe-client.bsc.gnosis.io/v1',
+  txServiceUrl: 'https://safe-transaction.bsc.gnosis.io/api/v1',
+  safeUrl: 'https://bsc.gnosis-safe.io/app',
+  safeAppsUrl: 'https://safe-apps-bsc.staging.gnosisdev.com',
+  gasPriceOracle: {
+    url: 'https://bscgas.info/gas',
+    gasParameter: 'standard',
+  },
+  rpcServiceUrl: 'https://bsc-dataseed.binance.org/',
+  networkExplorerName: 'BscScan',
+  networkExplorerUrl: 'https://www.bscscan.com/',
```

Comment:
This URL should not have a trailing slash because we add `/tx/0xXXX` in the code.

Reply:
Yes, I was even the one always going with this approach :sweat_smile: